app/src/ui/about/about.tsx

```diff
@@ -232,7 +237,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           </Row>
           <h2>{name}</h2>
           <p className='no-padding'>
-            Version {version} ({releaseNotesLink})
+            Version <span title='Click to copy' className='version-text' onClick={this.onClickVersion}>{version} <Octicon symbol={OcticonSymbol.clippy} /></span> ({releaseNotesLink})
```

Comment:
I think we should be consistent with how we set things on the clipboard throughout the app. I _think_ the work in #1778 is something we should also do here (use a link that copies to the clipboard) rather than an icon.